src/content/basics/nodepools/index.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Node Pools
+description: A general description of node pools as a concept, it's benefits, and some details you should be aware of.
+date: 2019-10-28
+weight: 15
+type: page
+categories: ["basics"]
+---
+
+# Node Pools
+
+<div class="well disclaimer">
+Node pools are a new concept to be introduced soon to Giant Swarm customers on AWS.
+</div>
+
+## Advantages
+
+Prior the introduction of node pools, a cluster could only comprise one type of worker node. The cluster would have
+to be scaled as a whole, and the availability zone distribution would apply to all worker nodes of a cluster. This
+would mean that every worker node would have to be big enough to run the largest possible workload, in terms of
+memory and CPU resources required. At the same time, all worker nodes in the cluster would have to use the same
+availability zone distribution, even if some workloads wouldn't require the increased availability that would come
+with using more availability zones.
+
+Node pools are independent groups of worker nodes belonging to a cluster, where all nodes within a pool share a
+common configuration. You can combine any sort of node pool within one cluster. Node pools can differ regarding:
+
+- EC 2 instance type
+- Availability zone distribution
+- Scaling configuration (number of nodes)
+
+A node pool is identified by a unique ID that is generated on creation and by a name that you can pick as a cluster
+administrator.
+
+## Lifecycle
+
+Node pools can be created when creating a cluster
+
+- via the Giant Swarm web interface ("happa")
+- via the CLI command [`gsctl create cluster`](/reference/gsctl/create-cluster/)
+
+or any time after the cluster has been created
+
+- via the Giant Swarm web interface
+- via the CLI command [`gsctl add nodepool`](/reference/gsctl/add-nodepool/)
+
+These tools also support modification of node pools and their deletion.
+
+Once a node pool has been created, as soon as the workers are available, they will
+join the cluster and appear in your `kubectl get nodes` listing. You can identify the
+nodes' node pool using the `giantswarm.io/machine-deployment` label.
+
+```nohighlight
+kubectl get nodes \
+  -o=jsonpath='{range .items[*]}{.metadata.labels.giantswarm\.io/machine-deployment}{"\t"}{.metadata.name}{"\n"}{end}' | sort
+```
+
+Some details of a node pool can be modified after creation:
+
+- The node pool name
+- The scaling range (min(max)
```

Comment:
The braces are weird. Not sure if this is some well established format.

Reply:
No, that's a typo. Will change to `(min/max)`.